starter_code/index.js

```diff
@@ -1 +1,26 @@
-const Elevator = require('./elevator.js');
+const Elevator = require("./elevator.js");
+const Person = require("./person.js");
+
+var elevator = new Elevator();
+
+elevator.floorUp();
+elevator.floorUp();
+
+elevator.update();
```

Comment:
the `update` method should be called for you in the logic. Here you should be calling `start` method, that starts your interval and calls update.